plugin.video.liveleak/addon.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.liveleak"
+    name="LiveLeak"
+    version="0.0.7"
+    provider-name="Botster">
+    <requires>
+        <import addon="xbmc.python" version="2.24.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Watch LiveLeak.com Videos</summary>
+        <description lang="en_GB">Redefining The Media</description>
+        <language>en_GB</language>
```

Comment:
Language should be en. The en_GB stuff is only for localizedstrings/translations(so, only for summary, description and translations if you had them). Language is for the video content and should be en. Sorry if my previous comment was confusing.